build.sbt

```diff
@@ -3,7 +3,7 @@ import uk.gov.hmrc.DefaultBuildSettings
 lazy val appName: String = "charities-registration-frontend"
 
 ThisBuild / majorVersion := 0
-ThisBuild / scalaVersion := "2.13.12"
+ThisBuild / scalaVersion := "2.13.14"
 
```

Comment:
Once all the changes in the [GovukLayoutWrapper](https://github.com/hmrc/charities-registration-frontend/blob/main/app/views/templates/GovukLayoutWrapper.scala.html) have been done, [line 32](https://github.com/hmrc/charities-registration-frontend/blob/main/build.sbt#L32) can be removed.